BnbnavNetClient/Models/Player.cs

```diff
@@ -1,7 +1,24 @@
-﻿namespace BnbnavNetClient.Models;
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Avalonia;
+using Avalonia.Animation;
+using Avalonia.Media;
+using BnbnavNetClient.Helpers;
+using BnbnavNetClient.Services;
+using Timer = System.Timers.Timer;
+
+namespace BnbnavNetClient.Models;
 
 public sealed class Player
 {
+    private readonly MapService _mapService;
+    private readonly Timer _timer;
+    private Mutex _lastSnapMutex = new();
```

Comment:
no explicit private, does the mutex need to not be readonly? (I haven't checked)